src/content/en/updates/2019/03/prefers-reduced-motion.md

```diff
@@ -0,0 +1,231 @@
+reveal animations are gone, and just the regular scrolling motion is left. The screencast below
+shows the demo in action:
+
+<video width="1098" height="870" intrinsicsize="1098x870" controls>
+  <source src="../../videos/2019/03/prefers-reduced-motion/prefers-reduced-motion.mp4" type="video/mp4">
+</video>
+
+## (Bonus) Forcing reduced motion on all websites
+
+Not every site will use `prefers-reduced-motion`, or maybe not consequently enough for your taste.
+If you, for whatever reason, want to stop motion on all websites, you actually can. One way to make
+this happen is to inject a stylesheet with the following CSS into every webpage you visit.
+There are several
+[browser extensions](https://chrome.google.com/webstore/search/user%20stylesheets?_category=extensions)
+out there (use at your own risk!) that allow for this.
+
+```CSS
+@media (prefers-reduced-motion: reduce) {
+  * {
+    animation-duration: 0.001s !important;
+    transition-duration: 0.001s !important;
+  }
+}
+```
+
+The way this works is that the CSS above overrides the durations of all animations and transitions
+to such a short time that they are not noticeable anymore. As some websites depend on an animation
+to be run in order to work correctly (maybe because a certain step depends on the firing of the
+[`animationend` event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/animationend_event)),
+the more radical `animation: none !important;` approach wouldn't work. Even the above hack is not
+guaranteed to succeed on all websites, so be sure to deactivate it when you notice breakage.
+
+## Conclusions
+
+Respecting user preferences is key for modern websites, and browsers expose more and more features
+to enable web developers to do so. The CSS Working Group are currently standardizing more
+[user preference media queries](https://drafts.csswg.org/mediaqueries-5/#mf-user-preferences)
+like `prefers-reduced-transparency` (detects if the user prefers reduced transparency),
+`prefers-contrast` (detects if the user has requested the system increase or decrease the amount of
+contrast between adjacent colors), `prefers-color-scheme` (detects if the user prefers a light or
+dark color scheme), and `prefers-reduced-transparency` (detects if the user prefers reduced
+transparency). Watch this space, we will let you know once they launch in Chrome!
+
+## Related Links
+
+- Latest Editor's Draft of the
+[Media Queries Level&nbsp;5](https://drafts.csswg.org/mediaqueries-5/#prefers-reduced-motion) spec.
+- `prefers-reduced-motion` on
+[Chrome Platform Status](https://www.chromestatus.com/feature/5597964353404928).
+- `prefers-reduced-motion` [Chromium bug](http://crbug.com/722548).
+- Blink
+[Intent to Implement posting](https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/NZ3c9d4ivA8/BIHFbOj6DAAJ).
+
+## Acknowledgements
+
+Massive shout-out to [Stephen McGruer](https://twitter.com/stephenmcgruer) who has implemented
```

Comment:
My twitter is essentially dead (as you can see, it is filled with random contest entries and nothing else ;)). I would rather this either link to my github (https://github.com/stephenmcgruer) or just no link at all is fine.

Reply:
I link to https://github.com/stephenmcgruer now :-D